src/components/BooksForm.js

```diff
@@ -33,25 +33,36 @@ function Form() {
   };
 
   return (
-    <form className="FORM" onSubmit={handleSubmit}>
-      <input
-        type="text"
-        id="title"
-        name="title"
-        required
-        value={state.title}
-        onChange={onChangeHandler}
-      />
-      <input
-        type="text"
-        id="author"
-        name="author"
-        required
-        value={state.author}
-        onChange={onChangeHandler}
-      />
-      <button type="submit">Add Book</button>
-    </form>
+    <div className="Add-container">
+      <h2 className="add-title">Add Your Book</h2>
+      <section>
+        <form className="add-book" onSubmit={handleSubmit}>
+          <input
+            type="text"
+            className="title-input"
+            id="title"
+            placeholder="Add Book Title"
+            name="title"
+            required
+            value={state.title}
+            onChange={onChangeHandler}
+          />
+          <input
+            type="text"
+            id="author"
+            className="author-input"
+            name="author"
+            placeholder="Add Book Author"
+            required
+            value={state.author}
+            onChange={onChangeHandler}
+          />
+          <button className="add-button" type="submit">
+            Add Book
+          </button>
```

Comment:
- [ ] 👍 Great job with the color selection! To make your design even closer to the intended look, kindly consider updating the font family and padding to match the design. This will ensure a consistent and visually appealing layout. Keep up the good work! 👍🎨
![image](https://github.com/Nuga0909/bookstore3/assets/55840999/c9e7f1bd-5a56-4e6d-8db4-d8b98110486a)